.github/workflows/ci.yml

```diff
@@ -160,6 +162,8 @@ jobs:
         uses: "networktocode/gh-action-setup-poetry-environment@v6"
         with:
           poetry-version: "1.8.5"
+      - name: "Pip install virtualenv to avoid issue with virtualenv --wheel"
```

Comment:
This fix is not needed here. It has been upstreamed into the action itself with the same version tag.